driver/src/main/java/org/neo4j/driver/internal/logging/ConsoleLogging.java

```diff
@@ -94,10 +94,10 @@ public String format( LogRecord record )
             StringBuilder builder = new StringBuilder( 1000 );
             builder.append( dateFormat.format( new Date( record.getMillis() ) ) );
             builder.append(" ");
-            builder.append("[").append(record.getLoggerName()).append("] ");
-            // builder.append( "[" ).append( record.getSourceClassName() ).append( "." );
-            // builder.append( record.getSourceMethodName() ).append( "] - " );
-            // builder.append( "[" ).append( record.getLevel() ).append( "] - " );
+//            builder.append("[").append(record.getLoggerName()).append("] ");
```

Comment:
Should this line be uncommented? Can we remove next 3 commented lines?

Reply:
This ConsoleLogging class is just for internal use purpose. I would prefer keep these commented there in case if we ever need to print more info to help debug.